tools/misbehaving-jmx-server/src/main/java/org/datadog/misbehavingjmxserver/App.java

```diff
@@ -56,21 +67,74 @@ public void overrideFromEnv() {
         if (val != null) {
             this.controlPort = Integer.parseInt(val);
         }
+        val = System.getenv("CONFIG_PATH");
+        if (val != null) {
+            this.config_path = val;
+            System.out.println("updated config_path from env: " + config_path);
+        }
+    }
+
+    public void initConfig() throws IOException {
+        appSpec = getConfig();
+        if (appSpec != null){
+            System.out.println("AppSpec from " + config_path + " is:\n" + appSpec);
+        } else {
+            System.out.println("AppSpec is null");
+            return;
+        }
+
+    }
+
+    public AppSpec getConfig () {
+        File f = new File(config_path);
+        String yamlPath = f.getAbsolutePath();
+        try{
+            FileInputStream yamlInputStream = new FileInputStream(yamlPath);
+            Yaml yaml = new Yaml(new Constructor(AppSpec.class));
+            AppSpec test = yaml.load(yamlInputStream);
+            return test;
```

Comment:
If it was successfully read, lets print a `log.info` saying "Read in config values from file at path X"